.github/workflows/msal-node-e2e.yml

```diff
@@ -33,12 +33,10 @@ jobs:
         with:
           client-id: ${{ secrets.TEST_CLIENT_ID }}
           tenant-id: ${{ secrets.TEST_TENANT_ID }}
-          subscription-id: ${{ secrets.TEST_SUBSCRIPTION_ID }}
+          allow-no-subscriptions: true
   
       - name: 'Run az commands'
-        run: |
-          az account show
-          az group list
+        run: az keyvault secret show --name ${{ secrets.TEST_SECRET_NAME }} --vault-name ${{ secrets.TEST_VAULT_NAME }}
```

Comment:
I suggest doing this in a separate PR. The OBO test with secret should be ready to go and provides good value as is.